### PR TITLE
CCTRI-2025: implemented conf token supporting

### PR DESCRIPTION
**Pull request by MaxMhonc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,74 +127,14 @@ zappa undeploy dev
...
-4. The command will also build the links to the SecureX Threat Response pages (in all
-available regions) with the corresponding module creation forms. Select the
-link corresponding to your SecureX Threat Response region. The form there will require
-you to enter both your Lambda's `URL` and your `JWT` (along with a unique name)
-to finally create your SecureX Threat Response module.
-
-That is it! Your Serverless Relay is ready to use! Congratulations!
+Now, the only thing left to do is to follow one of these URLs to navigate 
+to SecureX Threat Response page in your region and create the Palo Alto AutoFocus
+module using your Lambda's URL and Have I Been Pwned API key:
```
Comment (AlinaBlahun) [inline README.md:134]: Palo Alto AutoFocus

Comment (MaxMhonc) [reply README.md]: Done

**Review by AlinaBlahun — COMMENTED** [review]
Sorry for the late note, but looks like this code: 
```
   _ = get_api_key()
   _ = get_observables()
```
should be removed from endpoints with no implementation (deliberate,  respond) - it’s kind of extra work for no reason.

**Comment by AnastasiiaRozlyvan** [comment]
I support Alina's comment, and I wanted to add something about `DashboardTileDataSchema`. It is implemented in the same way as in the serverless relay (i.e. it will cause errors). It's worth either removing it or fixing it, I think.

**Review by AnastasiiaRozlyvan — APPROVED** [review]

**Review by AlinaBlahun — APPROVED** [review]

**Pull request merged**